os/arch/arm/src/amebasmart/amebasmart_spi.c

```diff
@@ -611,8 +612,14 @@ static int amebasmart_spi_lock(FAR struct spi_dev_s *dev, bool lock)
  *
  *
  ************************************************************************************/
-void amebasmart_spi0select(FAR struct spi_dev_s *dev, enum spi_dev_e devid, bool selected)
+void amebasmart_spi_select(FAR struct spi_dev_s *dev, enum spi_dev_e devid, bool selected)
```

Comment:
If we can use this for both spi0 and spi1, how about naming `gpio_cs` instead of `gpio_cs0`?

Reply:
I mean the member `gpio_cs0` in the `struct amebasmart_spidev_s`.

Reply:
gpio_cs0 is used to control slave 0 connected to the particular spi port. If a second slave is attached, we will add gpio_cs1 to select it. The naming is not related to the port.